app/services/onboarding.ts

```diff
@@ -138,11 +139,13 @@ class OnboardingViews extends ViewHandler<IOnboardingServiceState> {
       steps.push(ONBOARDING_STEPS()[EOnboardingSteps.ThemeSelector]);
     }
 
-    if (this.getServiceViews(UserService).isTwitchAuthed) {
+    if (userViews.isTwitchAuthed) {
       steps.push(ONBOARDING_STEPS()[EOnboardingSteps.Optimize]);
     }
 
-    steps.push(ONBOARDING_STEPS()[EOnboardingSteps.Prime]);
+    if (userViews.isLoggedIn && !userViews.isPrime) {
```

Comment:
Happy to see `getServiceViews` working well in practice.  Thanks for embracing the new paradigm.